### ISSUE TITLE
Change readme.md

### ISSUE THREAD

**Issue by AnupKumarPanwar** [TheAlgorithms/Scala#9]
The readme.md file of this repo appears to be specific for sorting algorithms only. The needs to be updated with suitable description. Comment for the suggestions.

And for algorithms description send PRs on [Algorithms-Explained](https://github.com/TheAlgorithms/Algorithms-Explainations) repo.

**Comment by EdgeCaseBerg** [comment]
Isn't it also including search's? It's got a [header for that here](https://github.com/TheAlgorithms/Scala#search-algorithms)

**Comment by AnupKumarPanwar** [comment]
Yeah but, the description of the algo is to be kept in a separate repo, ie, https://github.com/TheAlgorithms/Algorithms-Explainations .

The README.md should include the structure of this repo, like a tree of the directories and files.

**Comment by varunu28** [comment]
@AnupKumarPanwar As the project structure has changed, I would be updating the README accordingly.

You have a good suggestion of adding the structure in which algorithms are kept. I will start working on that

**Comment by sentenza** [comment]
@Panquesito7 could you please add the hacktoberfest tag to this repo? I would like to contribute to this project.

**Comment by Panquesito7** [comment]
> @Panquesito7 could you please add the hacktoberfest tag to this repo? I would like to contribute to this project.

I'm not sure if this project would be eligible for Hacktoberfest, as most of the DSA repositories are excluded. Let me talk to one of the Hacktoberfest organizers about that. Thanks. 🙂

**Comment by Panquesito7** [comment]
> > I'm not sure if this project would be eligible for Hacktoberfest, as most of the DSA repositories are excluded. Let me talk to one of the Hacktoberfest organizers about that. Thanks. 🙂
> 
> @Panquesito7 What's the difference between https://github.com/TheAlgorithms/Python and this project then? The only difference that I can think of is the programming language used to implement the algorithms. Nevertheless that repository uses the hacktoberfest tag.

I've talked to a Hacktoberfest manager, and they said they are participating in Hacktoberfest. I added the tag now. Feel free to make any PRs or contribute!
